bpf/cpu/rbperf.bpf.c

```diff
@@ -178,19 +159,17 @@ read_ruby_lineno(u64 pc, u64 body, RubyVersionOffsets *version_offsets) {
     }
 }
 
-static inline_method void
-read_frame(u64 pc, u64 body, RubyFrame *current_frame,
-           RubyVersionOffsets *version_offsets) {
+static inline_method void read_frame(u64 pc, u64 body, symbol_t *current_frame, RubyVersionOffsets *version_offsets) {
     u64 path_addr;
     u64 path;
     u64 label;
     u64 flags;
     int label_offset = version_offsets->label_offset;
 
     LOG("[debug] reading stack");
+    __builtin_memset((void *)current_frame, 0, sizeof(symbol_t));
```

Comment:
nit: perhaps we should expect to do this in the caller so this function doesn't have to deal with this? I assume you had a chance to test it?

We can land this as-is, but we should benchmark this as it could shift-costs, or even worse, decrease the overall performance of the unwinder

Reply:
I'll address this in subsequent PRs.